shell/src/main/java/alluxio/shell/command/TestCommand.java

```diff
@@ -167,11 +159,11 @@ public void run(CommandLine cl) throws AlluxioException, IOException {
         }
       } else {
         System.out.println("test takes 1 option, not 0. Usage: test [-d|-f|-e|-s|-z] <path>");
-        return;
```

Comment:
If it is no need to print error message, for test command is just used for shell programming? Instead, just return 1 is ok?